.clang-format

```diff
@@ -56,8 +54,7 @@ IncludeCategories:
     - Priority: 1
 
 IndentCaseLabels: false
-#Unsupported in VS-included ClangFormat 6.0 we'll see about VS2019's ClangFormat 7.0
-#IndentPPDirectives: BeforeHash
+IndentPPDirectives: BeforeHash
```

Comment:
I have no idea what this one does.

Reply:
Ref: https://clang.llvm.org/docs/ClangFormatStyleOptions.html
>IndentPPDirectives (PPDirectiveIndentStyle)
>
>    The preprocessor directive indenting style to use.
>
> Possible values:
>
- `PPDIS_None` (in configuration: `None`) Does not indent any directives.
```
        #if FOO
        #if BAR
        #include <foo>
        #endif
        #endif
```
- `PPDIS_AfterHash` (in configuration: `AfterHash`) Indents directives after the hash.
```
        #if FOO
        #  if BAR
        #    include <foo>
        #  endif
        #endif
```
- `PPDIS_BeforeHash` (in configuration: `BeforeHash`) Indents directives before the hash.
```
        #if FOO
          #if BAR
            #include <foo>
          #endif
        #endif
```